### PR TITLE
Docs: making docs clearer on subpath

### PR DESCRIPTION
**Pull request by tolzhabayev**
**What is this feature?**

Following up on https://github.com/grafana/grafana/pull/81658

**Why do we need this feature?**
This tutorial has been a source of confusion and multiple issues in the past, trying to make it clear that it's either - handle the sub path in the proxy OR set the `service_from_sub_path` to `true`. Most get confused and do both.

**Who is this feature for?**
Grafana Administrators in the wild
 
**Which issue(s) does this PR fix?**:

### CODE REVIEW

**Review by wbrowne — APPROVED** [review]
🚀

File: docs/sources/tutorials/run-grafana-behind-a-proxy/index.md [hunk 1]
```diff
@@ -34,23 +34,9 @@ domain = example.com
 
 - Restart Grafana for the new changes to take effect.
 
-You can also serve Grafana behind a _sub path_, such as `http://example.com/grafana`.
+## Configure Reverse Proxy
```
Comment (josmperez) [inline docs/sources/tutorials/run-grafana-behind-a-proxy/index.md:37]:
```suggestion
## Configure reverse proxy
```

File: docs/sources/tutorials/run-grafana-behind-a-proxy/index.md [hunk 2]
```diff
@@ -240,6 +219,17 @@ http:
           - url: http://192.168.30.10:3000
 ```
 
-## Summary
+## Alternative for serving Grafana under a sub path
+**Warning:** You only need this, if you do not handle the sub path serving via your reverse proxy configuration.
```
Comment (josmperez) [inline docs/sources/tutorials/run-grafana-behind-a-proxy/index.md:223]:
```suggestion
**Warning:** You only need this if you don't handle the sub path serving via your reverse proxy configuration.
```

File: docs/sources/tutorials/run-grafana-behind-a-proxy/index.md [hunk 3]
```diff
@@ -240,6 +219,17 @@ http:
...
+
+If you don't want or can not use the reverse proxy to handle serving Grafana from a _sub path_, you can set the config variable `server_from_sub_path` to `true`
```
Comment (josmperez) [inline docs/sources/tutorials/run-grafana-behind-a-proxy/index.md:225]:
```suggestion
If you don't want or can't use the reverse proxy to handle serving Grafana from a _sub path_, you can set the config variable `server_from_sub_path` to `true`.
```

File: docs/sources/tutorials/run-grafana-behind-a-proxy/index.md [hunk 4]
```diff
@@ -240,6 +219,17 @@ http:
...
+
+1. Include the sub path at the end of the `root_url`.
+2. Set `serve_from_sub_path` to `true`
```
Comment (josmperez) [inline docs/sources/tutorials/run-grafana-behind-a-proxy/index.md:228]:
```suggestion
2. Set `serve_from_sub_path` to `true`:
```

**Review by josmperez — APPROVED** [review]
A few nits, otherwise LGTM.

**Pull request merged**